week-9/Homework/mandatory/1-practice/2-code-reading.md

```diff
@@ -61,3 +64,4 @@ console.log(y);
 ```
 
 What will be the output of this code. Explain your answer in 50 words or less.
+ console.log(x) will display 9 on the screen as x is a global, while console.log(y) will display {x:10}  as val.x will refer to the same reference of x on object y then it will add 1 to it  as y is global.
```

Comment:
- correct.
- f1 will return 10.
- console.log x will return 9. 
- x is const 
-  y will log 10